src/Pos/Ssc/NistBeacon.hs

```diff
@@ -45,31 +46,30 @@ instance Ssc SscNistBeacon where
     type SscVerifyError SscNistBeacon = ()
 
     mkSscProof = Tagged $ const ()
-    sscCreateNodeContext = Tagged $ const (return ())
+    sscCreateNodeContext = Tagged $ const pass
 
 instance SscHelpersClass SscNistBeacon where
     sscVerifyPayload = Tagged $ const $ const $ Right ()
 
 instance SscWorkersClass SscNistBeacon where
-    sscWorkers = const (Tagged ([], mempty))
+    sscWorkers = Tagged ([], mempty)
     sscLrcConsumers = Tagged []
 
 instance SscListenersClass SscNistBeacon where
     sscListeners = return $ Tagged ([], mempty)
     sscStubListeners = Tagged ([], mempty)
 
 instance SscLocalDataClass SscNistBeacon where
-    sscGetLocalPayloadQ _ = pure ()
+    sscGetLocalPayloadQ _ = pass
     sscNormalizeU _ _ _ = pass
-    sscNewLocalData = pure ()
+    sscNewLocalData = pass
```

Comment:
I don't like some changes here. `sscNewLocalData` returns local data which is defined as `()`.
`pass` has a semantics of `do nothing`.
`pure ()` has a semantics of `return ()`.

It's a clumsy explanation, but I think you got it.